Kafka_Consumer.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
```

Comment:
Can you move Kafka_Consumer.py to examples/?   Can you also rename this to be "log_consumer.py"